app/models/translation.rb

```diff
@@ -95,18 +95,36 @@ def prevent_destroy_published
   def push_published_to_s3
     return unless is_published
 
-    name_desc_onesky if resource.uses_onesky?
+    phrases = manifest_translated_phrases
```

Comment:
this should probably be protected by `resources.uses_onesky?`

Reply:
Right, good call